tests/data/application-config-1.json

```diff
@@ -7,7 +7,9 @@
     "regular_users"
   ],
   "rates_urls": [
-    "http://arkdedicated.com/dynamicconfig.ini"
+    {
+      "url": "http://arkdedicated.com/dynamicconfig.ini"
+    }
```

Comment:
As highlighted above, you'll notice I got lazy here an only changed this from a string to a dict/object keyed with `url`, but it might be better to explicitly provide all other params too. Your call